templates/ecs/content/ecs.tf

```diff
@@ -0,0 +1,28 @@
+module "ecs" {
```

Comment:
Could you run `terraform fmt` on the *.tf files when you've got a chance.

Reply:
Done!